routes/user.js

```diff
@@ -0,0 +1,170 @@
+var sql = require('mysql');
+var md5 = require('md5');
+var connection = require('../modules/connection');
+var responses = require('../modules/responses');
+var comFunc = require('../modules/commonFunction');
+
+
+exports.login = function(req , res) {
+
+	var email = req.body.email;
+	var password = req.body.password;
+
+	var sql = "SELECT  * FROM `tb_user` WHERE `email`=? AND `password`=?";
+	var password = md5(password);
+	var values = [email,password];
+ 	 	
+	connection.query(sql, values, function(err, result){
+		if (err) {
+			responses.sendError(res);
+			return;
+		} else if(result.length>0)
+			{
+				result[0].password = "";
+				responses.resultview(result , res);
```

Comment:
use object destructring here, no need of result[0].password

Reply:
we didn't learn about this till now.

Reply:
google